### DISCUSSION TITLE
New install - dirs not writable?

### DISCUSSION THREAD

**Discussion by Chad-ll** [Kitware/CDash#2428] [Q&A] [answered]
Hi --

I am setting up a new CDash installation and have gotten as far as the` https://<server>/install.php` page, but it is giving me the following errors:

> Your backup directory is not writeable, make sure the web process can write into the directory.
> Your upload directory is not writeable, make sure the web process can write into the directory.

I'm running CDash 3.4.1, RHEL 8.10 httpd with php 8.2, and PostgreSQL 15.

I'm not seeing anything in CDash/storage/logs/<file>.log about non-writable directories and everything under CDash/ is `chmod g+rw`

What directories is it talking about and what permissions should they be?

Thanks!

**Comment by williamjallen** [comment, marked as answer]
Try running `php artisan storage:mkdirs` to automatically create these directories.

**Reply by Chad-ll** [reply]
That did it, thank you!
